transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditor.java

```diff
@@ -514,4 +530,68 @@ void updateHelpText() {
         }
         helpText.getParent().layout();
     }
+
+    public void updateManifestPackageImports(IProject project, File pomFile, IProgressMonitor monitor) throws Exception {
+        IPath pomPath = project.getRawLocation() != null
+                        ? project.getRawLocation()
+                        : ResourcesPlugin.getWorkspace().getRoot().getLocation().append(project.getFullPath());
+        Model pomModel = MavenPlugin.getMaven().readModel(new File(pomPath.append("pom.xml").toOSString()));  //$NON-NLS-1$
+        if ("war".equals(pomModel.getPackaging())){ //$NON-NLS-1$
+        	return; 
+        }
+        Build build = pomModel.getBuild();
+        Map<String, Plugin> pluginsByName = build.getPluginsAsMap();
+        Plugin plugin = pluginsByName.get("org.apache.felix:maven-bundle-plugin"); //$NON-NLS-1$
+        if (plugin == null) {
+            plugin = new Plugin();
+            plugin.setGroupId("org.apache.felix"); //$NON-NLS-1$
+            plugin.setArtifactId("maven-bundle-plugin"); //$NON-NLS-1$
+            plugin.setVersion("3.2.0"); //$NON-NLS-1$
+            plugin.setExtensions(true);
+            build.addPlugin(plugin);
+        }
+        Xpp3Dom config = (Xpp3Dom)plugin.getConfiguration();
```

Comment:
I find this unchecked cast problematic. Plugin.getConfiguration could be null or something different and then the type cast will break.